### ISSUE TITLE
Youtube embed inside column breaks whenever page is edited

### ISSUE THREAD

**Issue by thpglobal** [WordPress/gutenberg#24176]
**Describe the bug**
Youtube embeds inside a two-column block breaks

**To reproduce**
1. Edit a page
2. Insert a two-column block
3. Insert a youtube embed in each column
4. Save it and it works.
5. Go back into that page - edit some other part of it and save - youtubes disappear.
6. Go back into the editor - error message.

**Expected behavior**
Youtube embeds should not disappaer.

**Screenshots**

**Editor version (please complete the following information):**
- WordPress version: [e.g: 5.4.2]
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"] Default.
![Screenshot 2020-07-23 at 7 01 16 PM](https://user-images.githubusercontent.com/15740550/88348127-92905e00-cd1a-11ea-8855-02a56aed70a1.png)

- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
- To report a security issue, please visit the WordPress HackerOne program: https://hackerone.com/wordpress.

**Comment by ehti** [comment]
@thpglobal I tried to reproduce it but couldn't. Tried both with and without the Gutenberg plugin installed. Chrome.

1. Added two columns.
2. Then a YouTube video to each fo the columns
3. Saved/Published
4. Went back to posts list, opened the post again in the editor
5. Added more stuff to the post, updated. Videos were visible
6. Re-opened the editor, and could still see the videos

I suspect it could be coming from a different plugin you might installed. Does it happen on a default WordPress site with no 3rd party plugins?

Thanks
